### ISSUE TITLE
Upgrade Seapickle

### ISSUE THREAD

**Issue by YaNIND** [xSuperr/VanillaBlocks#42]
maybe you can use example in concretepowder

**Comment by xSuperr** [comment]
Please provide more context.

**Comment by YaNIND** [comment]
private function checkAdjacentWater() : ?Block{
		for($i = 1; $i < 6; ++$i){ //Do not check underneath
			if($this->getSide($i) instanceof Water){
				return BlockFactory::get(Block::CONCRETE, $this->meta);
			}
		}

		return null;
	}

**Comment by YaNIND** [comment]
trying this?

**Comment by xSuperr** [comment]
No, why is it needed? If you are talking about Sea Pickles only being able to be placed in water, check the waterlogging issue.
